### PR TITLE
Implement transaction repository, remove Squadron & update ci.yml

### PR DESCRIPTION
**Pull request by Korgath**
## Description

- Implement transaction repository
- Removed Transaction_is_Registered test
- Update ci.yml
- Removed Squadron

## References

* #0000

### CODE REVIEW

File: source/B2B.Transactions.Tests/TransactionHandlingTests.cs [hunk 1]
```diff
@@ -16,29 +16,18 @@
...
-        [Fact]
-        public async Task Transaction_is_registered()
-        {
-            var transaction = CreateTransaction();
-            await RegisterTransaction(transaction).ConfigureAwait(false);
-
-            var savedTransaction = _transactionRepository.GetById(transaction.Message.MessageId);
-            Assert.NotNull(savedTransaction);
-        }
-
```
Comment (kft) [inline source/B2B.Transactions.Tests/TransactionHandlingTests.cs:41]: Why is this test removed ?

Comment (Korgath) [reply source/B2B.Transactions.Tests/TransactionHandlingTests.cs:41]: All it does is test DB implementation?

File: source/B2B.Transactions/DataAccess/MarketRolesContext.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+using System;
+using B2B.Transactions.DataAccess.Transaction;
+using B2B.Transactions.Transactions;
+using Microsoft.EntityFrameworkCore;
+
+namespace B2B.Transactions.DataAccess
+{
+    public class MarketRolesContext : DbContext
+    {
```
Comment (kft) [inline source/B2B.Transactions/DataAccess/MarketRolesContext.cs:23]: How about another name than MarketRoles ? B2BContext or something like that

**Review by kft — CHANGES_REQUESTED** [review]
I do not see a new test replacing the removed one. Why is this test not valid anymore ?

File: source/B2B.Transactions/DataAccess/MarketRolesContext.cs (hunk 2 above)
Comment (Korgath) [reply source/B2B.Transactions/DataAccess/MarketRolesContext.cs]: Aah yes ofc

**Comment by Korgath** [comment]
As discussed with @kft - database implementation for testing will be implemented

File: source/B2B.Transactions.Tests/TransactionHandlingTests.cs [hunk 3]
```diff
@@ -37,6 +47,8 @@ public async Task Transaction_is_registered()
 
             var savedTransaction = _transactionRepository.GetById(transaction.Message.MessageId);
             Assert.NotNull(savedTransaction);
+
+            await DisposeDatabaseAsync().ConfigureAwait(false);
```
Comment (kft) [inline source/B2B.Transactions.Tests/TransactionHandlingTests.cs:51]: Can we handle this in Dispose instead ? Does not seem relevant for the tests

**Review by kft — APPROVED** [review]

**Pull request merged**